src/cairoWriter/writers/parameterListWriter.ts

```diff
@@ -30,13 +30,24 @@ export class ParameterListWriter extends CairoASTNodeWriter {
         this.ast,
         varTypeConversionContext,
       );
-      if (tp instanceof CairoDynArray && node.parent instanceof FunctionDefinition) {
-        return isExternallyVisible(node.parent) ||
-          node.getClosestParentByType(ContractDefinition)?.name.includes('@interface')
-          ? `${value.name}_len : ${tp.vLen.toString()}, ${value.name} : ${tp.vPtr.toString()}`
-          : `${value.name} : ${tp.toString()}`;
+      if (isReturnParamList(node)) {
+        // In Cairo1 return parameters are not named
+        if (tp instanceof CairoDynArray && node.parent instanceof FunctionDefinition) {
```

Comment:
it's used 3 times, but gets called 2. Refactor the code to use only 1 imo will make the code less readable. But I will do a minor refactor of the condition isReturnParamList inside the return instruction that I didn't do before because I think oversaturate the instruction.